prow/plugins.yaml

```diff
@@ -171,7 +172,7 @@ plugins:
     - welcome
     - wip
     - yuks
-  knative-prow-robot/test-infra:
```

Comment:
@upodroid You recently  updated robots. Is there anything else I need to do here? we should ensure that the bots are working off of infra.

Reply:
we should probably clean those up then